lib/ansible/config/manager.py

```diff
@@ -329,7 +329,10 @@ def _parse_config_file(self, cfile=None):
         ftype = get_config_type(cfile)
         if cfile is not None:
             if ftype == 'ini':
-                self._parsers[cfile] = configparser.ConfigParser()
+                kwargs = {}
+                if PY3:
+                    kwargs['inline_comment_prefixes'] = (';',)
```

Comment:
doesn't this also need to include `#`?

Reply:
Python2 only supports `;` for inline comment prefixes

> Configuration files may include comments, prefixed by specific characters (# and ;). Comments may appear on their own in an otherwise empty line, or may be entered in lines holding values or section names. In the latter case, they need to be preceded by a whitespace character to be recognized as a comment. (For backwards compatibility, only ; starts an inline comment, while # does not.)